test/axes/numericAxisTests.ts

```diff
@@ -431,4 +431,36 @@ describe("NumericAxis", () => {
     svg.remove();
   });
 
+  it("reasonably approximates tick label sizes with approximate measuring", () => {
+    var SVG_WIDTH = 500;
+    var SVG_HEIGHT = 100;
+
+    [[-1, 1],
+     [0, 10],
+     [0, 999999999]
+    ].forEach((domainBounds) => {
```

Comment:
I'm personally not a fan of operating on a literal as the above.  Can you wrap the array into a variable and then operate on the variable instead?